core/modules/qdisp/QueryResource.cc

```diff
@@ -56,8 +56,12 @@ void QueryResource::ProvisionDone(XrdSsiSession* s) { // Step 3
     LOGF_INFO("Provision done");
     if(!s) {
         // Check eInfo in resource for error details
-        int code;
-        char const* msg = eInfo.Get(code);
+        int code = -1;
+        char const* message = eInfo.Get(code);
+        std::string msg = "no message from XrdSsi, code may not be reliable";
```

Comment:
What does this message mean, code is the XrdSsi return code, the xrootd source code, or the czar source code? Could you please clarify that?